pytorch_lightning/trainer/training_loop.py

```diff
@@ -171,7 +171,7 @@ def setup_training(self, model: LightningModule):
         # if cluster resets state, the model will update with the saved weights
         self.trainer.model = model
 
-        # restore training and model before hpc is called
+        # restore model weights before hpc is called
```

Comment:
```suggestion
        # restore training state and model weights before hpc is called
```
this restores both model weights and training weights

Reply:
done